packages/sources/src/components/image/image.stories.ts

```diff
@@ -0,0 +1,78 @@
+import { bindTemplate, StorybookParameters } from '../../stories-helpers';
+
+import { imageArgsMapper, imageArgTypes } from './image.args';
+import { Image } from './image.models';
+
+export interface ImageParameters<TemplateFnReturnType> {
+  imageTemplate: (imageProperties: Image) => TemplateFnReturnType;
+}
+
+export function storiesOfImage<TemplateFnReturnType>(
+  {
+    module: mainModule,
+    storiesOf,
+    readme
+  }: StorybookParameters,
+  {
+    imageTemplate: imageTemplate
+  }: ImageParameters<TemplateFnReturnType>
+) {
+  const template = bindTemplate(imageArgsMapper, imageTemplate);
+
+  const stories = storiesOf('Image', mainModule)
+    .addParameters({
+      docs: {
+        page: readme
+      },
+      argTypes: imageArgTypes,
+      args: {
+        source: 'images/avatar.png',
+        alt: 'Alternatieve tekst'
```

Comment:
Onze voorbeelden moeten toegankelijk en representatief zijn: Zullen we `avatar.png` hernoemen naar `sneeuwpop.png` en dan "Sneeuwpop" als alt?

Reply:
En als je in `args` op component niveau een waarde zet hoeft deze in de `ArgType` niet als optioneel gemarkeerd te worden: In `ImageArgTypes` is `alt` met de `?` suffix gemarkeerd als optioneel.

Reply:
nee toch?

```
export interface ImageArgs {
  source: string;
  modifier?: string;
  alt: string;
}
```

Reply:
Ik heb wel de naam van `avatar.png` omgezet naar `sneeuwpop.png` en de alternatieve tekst aangepast naar `Afbeelding van sneeuwpop`

Reply:
Ik vergiste mij, ik dacht aan `modifier`. Het ziet er goed uit!

Naamgeving en alt tekst ok!